.github/workflows/build.yml

```diff
@@ -106,7 +99,7 @@ jobs:
             qt_ver: 6
             qt_host: mac
             qt_arch: ""
-            qt_version: "6.8.1"
+            qt_version: "6.7.3"
```

Comment:
Seems https://github.com/PrismLauncher/PrismLauncher/pull/3609 was squashed in here

Reply:
solved